### PR TITLE
Adding radial profiles to HAL

### PR DESCRIPTION
**Pull request by torresramiro350**
I have implemented a working example of radial profiles in HAL. The function follows the same principle as in the 3ML HAWCLike.py plugin without the dependence on LiFF functions. The function  get_excess_background picks out the active HealPix pixels at different radial bins and calculates the signal, background and model counts at each bin. The function get_radial_profile takes care of the 'ring' excess calculations. Lastly, the plot_radial_profiles takes care of the plotting. The arguments are the same as in the HAWCLike plugin. I've include a testing script to ensure it works accordingly.

### CODE REVIEW

**Review by xiaojieww — COMMENTED** [review]
Don't understand why need to change the installation. Could you tell me the reason why you need this change?

**Comment by xiaojieww** [comment]
Also, I noticed that this branch is 17 commits behind `threeML:master`, could you update it first?  @torresramiro350

**Comment by torresramiro350** [comment]
the branch has been updated. let me know if you experience any other issues.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Then:
 
 ```bash
 > pip install --no-binary :all: root_numpy 
-> pip uninstall hawc_hal -y ; pip install git+https://github.com/threeml/hawc_hal.git
+> pip uninstall hawc_hal -y ; pip install git+https://github.com/torresramiro350/hawc_hal.git
```
Comment (henrikef) [inline README.md]: Could you change this back to installing from the official repo, please?

Comment (torresramiro350) [reply README.md]: Done.

**Comment by henrikef** [comment]
@xiaojieww @torresramiro350 What's the status of this PR? Any remaining issues?

@torresramiro350 could you push some trivial change (even adding whitespace to the Readme should work) to trigger the automated tests?

**Comment by xiaojieww** [comment]
@torresramiro350 I apologize for the slow response. @henrikef Some tests with astromodels didn't succeed, should we worry about this? Not sure if this is related to recent updates?

**Comment by henrikef** [comment]
You will need to merge #71 first (and then merge the new master into this branch). I just merged the current master into #71 and restarted the tests.

**Comment by xiaojieww** [comment]
> #71

After the tests are done, I will merge the #71 first, then go back here.

**Comment by xiaojieww** [comment]
@torresramiro350 Can you merge the new master into your branch and then trigger the test? I just merged the #71 , it included propagating changes in astromodels interface to HAL.

**Comment by torresramiro350** [comment]
@xiaojieww Done.

**Comment by xiaojieww** [comment]
> 

It still has three checks that didn't pass the check. Seems they had the same checks failed before I merge #71, but these tests were passed in #71.   Do you @henrikef  and @torresramiro350 have any ideas?

**Comment by henrikef** [comment]
Well it looks like it goes into some infinite loop/wait cycle and the test gets canceled after some time. Can you reproduce the issue locally on a mac computer?

**Comment by xiaojieww** [comment]
> Well it looks like it goes into some infinite loop/wait cycle and the test gets canceled after some time. Can you reproduce the issue locally on a mac computer?

Sorry I missed this comment. For the `threeML` and `astromodels` test without HAL installation, it passed on Mac. We can see this in #78 as well. All the tests passed on Mac but the CI test failed for mac.

**Review by xiaojieww — APPROVED** [review]

**Pull request merged**